automation/src/test/scala/org/broadinstitute/dsde/firecloud/test/security/AuthDomainSpec.scala

```diff
@@ -28,169 +30,409 @@ class AuthDomainSpec extends FreeSpec /*with ParallelTestExecution*/ with Matche
 
 
   "A workspace with an authorization domain" - {
+    "with one group inside of it" - {
+      "can be created" in withWebDriver { implicit driver =>
+        withGroup("AuthDomainSpec") { authDomainName =>
+          withCleanUp {
+            val workspaceListPage = signIn(Config.Users.fred)
 
-    "can be created by a user who is in a managed group" in withWebDriver { implicit driver =>
-      withGroup("AuthDomainSpec") { authDomainName =>
-        withCleanUp {
-          val workspaceListPage = signIn(Config.Users.fred)
-
-          val workspaceName = "AuthDomainSpec_create_" + randomUuid
-          register cleanUp api.workspaces.delete(projectName, workspaceName)
-          val workspaceDetailPage = workspaceListPage.createWorkspace(projectName, workspaceName, Option(authDomainName)).awaitLoaded()
+            val workspaceName = "AuthDomainSpec_create_" + randomUuid
+            register cleanUp api.workspaces.delete(projectName, workspaceName)
+            val workspaceSummaryPage = workspaceListPage.createWorkspace(projectName, workspaceName, Set(authDomainName)).awaitLoaded()
 
-          workspaceDetailPage.ui.readAuthDomainGroups should include (authDomainName)
+            workspaceSummaryPage.ui.readAuthDomainGroups should include(authDomainName)
 
-          workspaceListPage.open.filter(workspaceName)
-          workspaceListPage.ui.looksRestricted(projectName, workspaceName) shouldEqual true
+            workspaceListPage.open.filter(workspaceName)
+            workspaceListPage.ui.looksRestricted(projectName, workspaceName) shouldEqual true
+          }
         }
       }
-    }
 
-    "when not shared with a user who is not in the authorization domain" - {
+      "can be cloned and retain the auth domain" in withWebDriver { implicit driver =>
+        withGroup("AuthDomainSpec", List(Config.Users.george.email)) { authDomainName =>
+          withWorkspace(projectName, "AuthDomainSpec_share", Set(authDomainName)) { workspaceName =>
+            withCleanUp {
+              api.workspaces.updateAcl(projectName, workspaceName,
```

Comment:
you did- i had updated that case but failed to update the 5 other places where this was done. i've taken care of all of them now